provisioning/setup.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# Setup utility for Hare to configure Hare related settings, e.g. logrotate,
+# report unsupported features, etc.
+
+import argparse
+import asyncio
+import json
+import logging
+import shutil
+import subprocess
+
+from eos.utils.product_features import unsupported_features
+
+
+def get_data_from_provisioner_cli(method, output_format="json") -> str:
+    try:
+        process = subprocess.run(['provisioner', f'{method}', f'--out={output_format}'],
+                                 check=True,
+                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                                 shell=False)
+        stdout = process.stdout
+        rc = process.returncode
+    except Exception as e:
+        logging.error('Failed to fetch data from provisioner (%s)', e)
+        return 'unknown'
+    if rc != 0:
+        return 'unknown'
+    res = stdout.decode('utf-8')
+    return json.loads(res)['ret'] if res else 'unknown'
+
+
+def logrotate_config():
+    try:
+        setup_info = get_data_from_provisioner_cli('get_setup_info')
+        if setup_info != 'unknown':
```

Comment:
`setup_info` variable is not needed.
```python
        if get_data_from_provisioner_cli('get_setup_info') != 'unknown':
            ...
```

Reply:
But it is referenced on the next line,
> server_type = setup_info['server_type']

Reply:
It doesn't have to. 🤓
```python
        if get_data_from_provisioner_cli('get_setup_info') != 'unknown':
            ...
```
See? Equivalent code, no variable.

Reply:
Seems like something terribly wrong with my vision but I am not getting then from where do we get the `server_type` , we we do not save the output of `get_data_from_provisioner_cli`, we invoke `get_data_from_provisioner_cli` again?

Reply:
@mssawant No, it's not your vision, it's mine being unfocused.  Sorry for the noise.